@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -690,6 +690,36 @@ export class TileGeometryCreator {
                 const hasSolidLinesOutlines: boolean =
                     isSolidLineTechnique(technique) && technique.secondaryWidth !== undefined;
 
+                const theme = mapView.theme;
```

Comment:
I think this logically belongs to decoder.
Even if it's quite cheap, there is no reason to this in this now.

But it requires `Theme` access in decoder ? 
Shall we create another PR that brings theme back to decoder ?

Reply:
yep, we can't do that in the decoder because we don't have access to theme there.